website/azurerm.erb

```diff
@@ -1248,6 +1248,15 @@
               </ul>
             </li>
 
+            <li>
```

Comment:
Is there a way to generate these sections via `make`? I did this one manually and it appeared on `make website`